pylint/checkers/classes/special_methods_checker.py

```diff
@@ -222,7 +222,8 @@ def _check_unexpected_method_signature(self, node: nodes.FunctionDef) -> None:
             emit = mandatory not in expected_params
             # mypy thinks that expected_params has type tuple[int, int] | int | None
             # But at this point it must be 'tuple[int, int]' because of the type check
-            expected_params = f"between {expected_params[0]} or {expected_params[1]}"  # type: ignore[assignment]
+            expected_params = f"between {expected_params[0]} or \
```

Comment:
Please no backlashes :)

Reply:
Ok, I changed the code so that it works with the length parameter